{{cookiecutter.project_slug}}/_/deployment/python/deployment/application/base/cronjob.yaml

```diff
@@ -18,11 +18,20 @@ spec:
             command: ['/bin/bash', '-c']
             args:
               - python manage.py dumpdata -o backups/db.json ;
-              - cat backups/db.json
+                echo backups/db.json ;
```

Comment:
What's the idea behind the `echo backups/db.json`?

Reply:
I wanted to add an echo so that I could be sure that some command was run.  In one of my tests, there was no error, but nothing in the log/output at all, so I wasn't sure that any command had run.